README.Rmd

```diff
@@ -26,6 +26,33 @@ library(magrittr)
 library(data.table)
 library(ggplot2)
 library(knitr)
+
+## code to determine version inspired by [pkgdown:::dev_mode_auto()]
```

Comment:
This is a nice idea! Language wise it might be a good idea to talk about the release and unreleased version or similar vs CRAN version.

Reply:
Not sure - I suspect people might know whether they've installed from CRAN or GitHub but possibly not what we mean by release/unreleased.

Reply:
Though technically it's incorrect given that you can also install the CRAN version from GitHub etc.

Reply:
Went for stable/development in 450492da9d4ea995e809d333f0d50dcb0c9d4d34